### PR TITLE
Break out scripts to install and setup

### PR DESCRIPTION
**Pull request by pxp928**
Signed-off-by: pxp928 <parth.psu@gmail.com>

Breakout the scripts based on issue #292 and the discussion around reorg. Break out scripts to install and setup to allow for ease of install and development.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ setup-minikube: ## Setup a Kubernetes cluster using Minikube
 	bash platform/00-kubernetes-minikube-setup.sh
 
 .PHONY: setup-frsca
-setup-frsca: setup-certs setup-registry setup-tekton-chains setup-spire setup-vault
+setup-frsca: setup-certs setup-registry install-tekton-pipelines setup-tekton-pipelines install-tekton-chains setup-tekton-chains install-spire setup-spire install-vault setup-vault
```
Comment (sudo-bmitch) [inline Makefile:36]: Should kyverno be part of frsca? Seems like we'd want it here instead of adding entries to the quickstart.

**Review by sudo-bmitch — COMMENTED** [review]
Just a quick comment without a detailed look at the other changes. We probably want to document the general stages for file numbering, and whether we can use the same number for different scripts that can be run concurrently.

File: Makefile (hunk 1 above)
Comment (pxp928) [reply Makefile:36]: I agree that it should be added. For some reason we did not have it as part of `setup-frsca` so I did not move it. Maybe it was a remanent of the old work, but I will add it in.

**Comment by pxp928** [comment]
> Just a quick comment without a detailed look at the other changes. We probably want to document the general stages for file numbering, and whether we can use the same number for different scripts that can be run concurrently.

Need to think about this. Need to do some tests to see what can come up concurrently.

**Comment by pxp928** [comment]
Updated `setup-frsca` to install scripts in parallel as @sudo-bmitch suggested.

**Review by bradbeck — APPROVED** [review]

**Review by sudo-bmitch — APPROVED** [review]
LGTM. There's still some stuff I want to change, but I'm adding that to my list to send as a separate PR.

**Pull request merged**